charts/shoot-core/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -75,3 +91,5 @@ spec:
       - name: vpn-shoot-dh
         secret:
           secretName: vpn-shoot-dh
+      - name: shared-init-config
+        emptyDir: {}
```

Comment:
I think you should add `emptyDir` to the list of allowed `volumes` for the vpn-shoot pod security policy

Reply:
Good point, thanks for catching it. Done.